frontend/src/Signup.js

```diff
@@ -0,0 +1,53 @@
+import styles from './auth.module.css';
+import classNames from 'classnames';
+// Sign up page
+function Signup() {
+  return (
+    <div className = {styles.auth}>
+      
+      <main className = {styles.main}>
+        <div className = {styles.border}>
+            <header className = {styles.app}>
+                <a href="./"><img src="career_connect_logo.png" alt="logo" width ="200" hieght="200" /></a>
+                <br />
+                    Start connecting today!
+                <br />
+            </header>
+            <br />
+            <header className = {styles.describeAuth}>Create an account</header>
+            <br/>
+            <item className = {styles.subheaders}>
+                Email
+            </item>
```

Comment:
There is no HTML element called `<item>`, if you're trying to create a header for an input field try using the `<label>` tag

```css
<label htmlFor="email" className={styles.subheaders}>Email</label>
```